How-To-Host-Feed.md

```diff
@@ -332,11 +332,11 @@ See https://help.sonatype.com/repomanager3/high-availability-introduction for de
 #### ProGet
 
 * Windows Server
-* CPU - 4 cores (Recommend more)
-* RAM - 8GB+
-* Storage (HDD) - We recommend 2-5TB of free space for a local file store of artifacts (default install needs 20GB).
-* Database: SQL Server. Assume high specs for SQL Server, pretty much the highest specs you can configure on a Windows machine. 16+ cores (CPU), 64GB+ RAM, 5TB+, etc
-* Chocolatey Repository Type: Chocolatey or NuGet
+* CPU - 2 cores (4+ cores recommended for large instances)
+* RAM - 4GB+ (8GB recommended for large instances)
+* Storage (HDD) - We recommend 2-5TB of free space for a local file store of artifacts (default install needs 10GB).
+* Database: Database: SQL Server Express or better. Install available during download. Minimum specs 1GB RAM, 6GB disk space, higher specs may be required if creating more than a simple feed repository.
```

Comment:
@scott64bt as @ferventcoder mentioned in your other PR, can you please remove the mention of SQL Express here?